mender.go

```diff
@@ -315,7 +315,7 @@ func (m *mender) doBootstrap() menderError {
 }
 
 func (m *mender) FetchUpdate(url string) (io.ReadCloser, int64, error) {
-	return m.updater.FetchUpdate(m.api.Request(m.authToken), url)
+	return m.updater.FetchUpdate(m.api, url)
```

Comment:
I agree with the fix, but double checking with @maciejmrowiec. Is it correct that we want no authorization when fetching the image? Some images could be slightly sensitive I suppose?